config/tekton-dashboard-deployment.yaml

```diff
@@ -32,3 +32,7 @@ spec:
         env:
         - name: PORT
           value: "9097"
+        - name: INSTALL_NAMESPACE
```

Comment:
should this be `INSTALLED_NAMESPACE` to match https://github.com/tektoncd/dashboard/pull/34/files#diff-51aee488c67e4d9a10cd51b3c5c22e59R78 ?

Reply:
This value is set automatically so it doesn’t need to be set manually.

Reply:
If this is redundant it should be removed to avoid confusion. This is defining an environment variable `INSTALL_NAMESPACE` that's not actually used anywhere and could be easily confused for the `INSTALLED_NAMESPACE` we're expecting?

Reply:
Yeah, I really don't think it is redundant, PR here. https://github.com/tektoncd/dashboard/pull/77

@akihikokuroda can you please take a look - in the description I've mentioned what I think, lemme know if I'm mistaken please!

Reply:
My bad.  Yes, you are right.